<ui version="4.0" >
 <class>UIDesktopTools</class>
 <widget class="QDialog" name="UIDesktopTools" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Desktop Applications</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <property name="horizontalSpacing" >
    <number>3</number>
   </property>
   <property name="verticalSpacing" >
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="4" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>3</number>
     </property>
     <property name="verticalSpacing" >
      <number>3</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="lNameFilter" >
       <property name="text" >
        <string>&amp;Name Filter :</string>
       </property>
       <property name="buddy" >
        <cstring>leNameFilter</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="leNameFilter" >
       <property name="toolTip" >
        <string>You can write there a filter for your application name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QLabel" name="lCategoriesFilters" >
       <property name="text" >
        <string>&amp;Categories Filters :</string>
       </property>
       <property name="buddy" >
        <cstring>leCategoriesFilters</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="leCategoriesFilters" >
       <property name="toolTip" >
        <string>You can search for application base on their desktop categories ( Qt, audio and more ), you need to separate each category by a semi colon ;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="6" row="1" column="0" >
    <widget class="QTreeWidget" name="twLeft" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item rowspan="2" row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="6" row="1" column="2" >
    <widget class="QListWidget" name="lwRight" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="pbUp" >
     <property name="text" >
      <string>&amp;Up</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/up.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="pbDown" >
     <property name="text" >
      <string>&amp;Down</string>
     </property>
     <property name="icon" >
      <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/down.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QToolButton" name="tbRight" >
     <property name="text" >
      <string>>></string>
     </property>
     <property name="shortcut" >
      <string>Ctrl++</string>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="3" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>299</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <widget class="QToolButton" name="tbLeft" >
     <property name="text" >
      <string>&lt;&lt;</string>
     </property>
     <property name="shortcut" >
      <string>Ctrl+-</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QProgressBar" name="pbLoading" >
       <property name="minimumSize" >
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="format" >
        <string>Loading Applications... %p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UIDesktopTools</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>633</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel" >
     <x>316</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UIDesktopTools</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>633</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel" >
     <x>404</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
